src/App.tsx

```diff
@@ -321,7 +421,15 @@ class ControlButtons extends React.Component<ControlProps, {}> {
         return (
             <div className="button-col">
                 <Button
-                    onClick={this.props.numpadMode !== "normal" ? this.props.onClickMode : () => {}}
+                    onClick={this.props.numpadMode !== "normal" ?
+                        (event) => {
+                            event.preventDefault();
+                            this.props.onClickMode();
+                        } :
+                        (event) => {
+                            event.preventDefault();
+                        }
+                    }
```

Comment:
This is   N O T   preventingDefault. I don't understand how the Square functions are preveting it properly but these functions are not . or somehow the globalMouseDown listener sees the event a defaultprevented when its coming from the Square listeners but not from this listener